web/client/stores/defaultOptions.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { CHANGE_BROWSER_PROPERTIES } from '../actions/browser';
+
+import layersEpics from '../epics/layers';
+import controlsEpics from '../epics/controls';
+import * as configEpics from '../epics/config';
+import timeManagerEpics from '../epics/dimension';
+
+import layers from '../reducers/layers';
+import mapConfig from '../reducers/config';
+import controls from '../reducers/controls';
+import theme from '../reducers/theme';
+import help from '../reducers/help';
+import mapReducer from '../reducers/map';
+
+import { mapConfigHistory, createHistory } from '../utils/MapHistoryUtils';
+import { splitMapAndLayers } from '../utils/LayersUtils';
+
+const map = mapConfigHistory(mapReducer);
+
+export const standardReducers = {
+    controls,
+    theme,
+    help,
+    map: () => null,
+    mapInitialConfig: () => null,
+    mapConfigRawData: () => null,
+    layers: () => null
+};
+
+export const standardEpics = {
+    ...layersEpics,
+    ...controlsEpics,
+    ...timeManagerEpics,
+    ...configEpics
+};
+
+export const standardRootReducerFunc = ({
```

Comment:
I think we should document this. What are the role of mapInitialConfig and mapConfigRawData? 
```suggestion
/**
 * default `rootReducerFunc` for MapStore
 * - Adds history for map (separating `map` from `layers` state)
 * - Adds `mapInitialConfig`
 * - Allow to override the state with `mapInitialConfig` and `mapConfigRawData`
 */
export const standardRootReducerFunc = ({
```

Reply:
I don't know the role of those reducers but I found out where they come from:

- `mapConfigRawData` has been added in this PR #4034 Confirmation dialog in map on user logout (https://github.com/geosolutions-it/MapStore2/pull/4581)

- `mapInitialConfig` is there from the beginning PR App component (https://github.com/geosolutions-it/MapStore2/pull/614)

Reply:
@mbarto Can you specify the role of them, if you remember, so we can document them? ( Don't know if this is the correct place to document the role of these configs, please suggest if you have a better idea )